docs/debugging-python-lambda-functions.md

```diff
@@ -0,0 +1,92 @@
+# Debugging Python lambda functions
+
+## Install and configure prerequisites
+
+1. Install [Docker](https://www.docker.com/get-started) and start the Docker service.
+2. Install [AWS SAM CLI](https://github.com/awslabs/aws-sam-cli/releases).
+3. Install [Python](https://www.python.org/downloads/). AWS Lambda supports Python 2.7, 3.6, and 3.7.
+4. Install [Visual Studio Code](https://code.visualstudio.com/download).
+5. Install the AWS Toolkit for Visual Studio Code. <!-- TODO: Add marketplace link once the toolkit is published. -->
+6. Install the [Python extension for Visual Studio Code](https://marketplace.visualstudio.com/items?itemName=ms-python.python).
```

Comment:
How about:

Install the [Python extension for Visual Studio Code](https://marketplace.visualstudio.com/items?itemName=ms-python.python). This extension gives VS Code the ability to debug Python applications.